app/views/shared/_sidebar_navigation.html.erb

```diff
@@ -1,13 +1,13 @@
-<% if local_assigns.has_key?(:tasklist) && show_tasklist_sidebar? %>
-  <%= render 'shared/sidebar_tasklist', tasklist: tasklist %>
-<% elsif @navigation.should_present_taxonomy_navigation? -%>
-  <%# FIXME: Markup can wrap all three sidebar navigation
-      once we've migrated the appropriate formats %>
-  <div class="column-third add-title-margin">
+<div class="column-third">
+  <% if local_assigns.has_key?(:tasklist) -%>
+    <% tasklist = local_assigns[:tasklist] -%>
+    <h2 class="task-list-title">
+       <a href="<%= tasklist[:base_path] %>"><%= tasklist[:title] %></a>
+    </h2>
+    <%= render 'govuk_component/task_list', tasklist[:tasklist] %>
+  <% elsif @navigation.should_present_taxonomy_navigation? -%>
```

Comment:
I've noticed the [tasklist page](https://government-frontend-pr-558.herokuapp.com/government/publications/car-show-me-tell-me-vehicle-safety-questions) doesn't display the tasklist header in the universal layout but does on live. Is this intentional?

**Tasklist Header:**
<img width="1023" alt="screen shot 2017-12-13 at 10 03 55" src="https://user-images.githubusercontent.com/29889908/33933521-e6135ee6-dfed-11e7-8520-ecf13c4a5f12.png">